src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -57,6 +57,35 @@ public Workspace(Version powerShellVersion, ILogger logger)
 
         #region Public Methods
 
+        /// <summary>
+        /// Creates a new ScriptFile instance which is identified by the given file
+        /// path and initially contains the given buffer contents.
+        /// </summary>
+        /// <param name="filePath">The file path for which a buffer will be retrieved.</param>
+        /// <param name="initialBuffer">The initial buffer contents if there is not an existing ScriptFile for this path.</param>
+        /// <returns>A ScriptFile instance for the specified path.</returns>
+        public ScriptFile CreateScriptFileFromFileBuffer(string filePath, string initialBuffer)
+        {
+            Validate.IsNotNullOrEmptyString("filePath", filePath);
+
+            // Resolve the full file path
+            string resolvedFilePath = this.ResolveFilePath(filePath);
+            string keyName = resolvedFilePath.ToLower();
```

Comment:
Should be `ToLowerInvariant` yeah?  And do we do this conditionally based on platform yet? Or is that only in 2.0?

Reply:
Yeah I can change it. But I did grab it from the current implementation of GetFile. Did you change it in 2.0?

Reply:
Ah just keep it the same then.